dataplane/forwarding/info.go

```diff
@@ -290,6 +291,11 @@ func (l *InfoList) lookupInfo(e *entry, arg []byte, l2 fwdpb.PacketHeaderId) (st
 	if err != nil {
 		return "", fmt.Errorf("infolist: Unable to create packet, err %v", err)
 	}
+	// By default, the packet logging is dependent on the --v flag. Override this behavior
```

Comment:
Is this temporary or always the case in the future?

Reply:
for this RPC we want maximum verbosity since its only for debugging